### ISSUE TITLE
nvcc fatal   : Unsupported gpu architecture 'compute_60'

### ISSUE THREAD

**Issue by uzeful** [pytorch/pytorch#2283]
After updating the latest pytorch, I encountered the above problem during installation. After checking the commits, I noticed there was indeed a revision to fix this problem. But why does this problem still occurs?

My configuration:
Ubuntu14.04
Cuda7.5+cudnn5

**Comment by soumith** [comment]
you need to upgrade to cuda8 for your gpu. it is new enough that cuda 7.5 does not support it.

**Comment by uzeful** [comment]
Thanks for your quick reply. 
By the way, I asked another question on https://discuss.pytorch.org/t/how-to-optimize-a-model-once-after-several-times-of-small-batchs-training/4505/1.

The following is your answer. And I want to ask whether the backward() function will average the gradient? 

num_batches = 0
for sample, target in dataset:
    out = model(sample)
    loss = loss_fn(out, target)
    loss.backward()
    num_batches += 1
    if num_batches == 10: # optimize every 10 mini-batches
        optimizer.step()
        model.zero_grad() # or optimizer.zero_grad()
        num_batches = 0
